### PR TITLE
Native ingester cms

### PR DESCRIPTION
**Pull request by kovacshuni**

### CODE REVIEW

**Comment by scott-ace-newton** [comment]
:+1:

File: services.yaml [hunk 1]
```diff
@@ -85,12 +85,12 @@ services:
...
   count: 2
 - name: native-ingester-cms@.service
-  version: v1.0.10
+  version: v1.0.3
   count: 2
 - name: native-ingester-metadata-sidekick@.service
   count: 2
 - name: native-ingester-metadata@.service
-  version: v1.0.10
+  version: v1.0.3
```
Comment (nwrigh) [inline services.yaml]: Is this odd version number jump to sync with the up-registry version?

Comment (kovacshuni) [inline services.yaml]: I've moved from up-registry to dockerhub. The version i started with was just a pure arbitrary choosing. In up-registry we have something like fourty... Don't worry about it.

**Comment by nwrigh** [comment]
:+1:

**Comment by BerniVarga** [comment]
+1

**Pull request merged**